### PR TITLE
Add complete tutorial to the documentation

### PR DESCRIPTION
**Pull request by xadupre**

### CODE REVIEW

File: .azure-pipelines/linux-CI-nightly.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ jobs:
 
 - job: 'Test'
   pool:
-    vmImage: 'Ubuntu-16.04'
+    vmImage: 'ubuntu-latest'
```
Comment (wenbingl) [inline .azure-pipelines/linux-CI-nightly.yml:13]: can you separate these CI pipeline changes into another PR?

Comment (xadupre) [reply .azure-pipelines/linux-CI-nightly.yml]: Done for all CI files.

Comment (xadupre) [reply .azure-pipelines/linux-CI-nightly.yml]: But the PR fails now so either I fix the examples which fails for ubuntu 16.04 or I write on the example it works for ubuntu 18.04.

Comment (wenbingl) [reply .azure-pipelines/linux-CI-nightly.yml]: It looks the failure happening on the more recent ONNXRuntime build. was it caused by the new ORT not supporting ubuntu 16.04 any more?

Comment (xadupre) [reply .azure-pipelines/linux-CI-nightly.yml]: I'm trying to make it work for that distribution.

Comment (xadupre) [reply .azure-pipelines/linux-CI-nightly.yml]: Fixed.

**Review by wenbingl — APPROVED** [review]

**Pull request merged**